src/js/components/pages/AccountPage.jsx

```diff
@@ -19,6 +20,34 @@ import SubscriptionBox from '../cards/SubscriptionBox';
 import ShareButton from '../ShareButton';
 import { addImageCredit } from '../../base/components/AboutPage';
 import Roles from '../../base/Roles';
+import TabsForGoodSettings from './TabsForGoodSettings';
+
+const Account = () => {
+	let xids = getAllXIds();
+	return <>
+		<div className="w-75 mx-auto">
+			<div className="text-center">
+				<h2>What to do now?</h2>
+				<p>You have already made the first important step in helping us: you joined our community. But there is more you can do!</p>
+			</div>
+			<MoreToDo xids={xids} />
+		</div>
+		
+		<h2 className="text-center mb-5">Your settings</h2>
+		<Settings xids={xids}/>
+	</>;
+};
+
+const tabs = {
+	account: {
+		content: <Account/>,
+		name: "My Account"
+	},
+	tabsForGood: {
+		content: <TabsForGoodSettings/>,
+		name: "Tabs for Good"
+	},
+};
```

Comment:
This kind of local ad-hoc data+code structure makes for confusing code. 
Instead, try a list of tab names for the sidebar, and then a `switch(name)` when you need to make the jsx.

Reply:
Refactored